pkgs/applications/virtualization/xenserver-buildroot/default.nix

```diff
@@ -0,0 +1,54 @@
+{stdenv, fetchurl, fetchgit, nix-template-rpm }:
+
+# Xenserver buildroot of somehow around XenServer 6.5.
+# Xenserver is not an application by itself, but just a collection of components.
+# This package only provides the build inputs to actually build the components of xenserver.
+stdenv.mkDerivation rec {
+  name = "xenserver-${version}";
+  version = "2015012901";
+
+  src = fetchgit {
+    url = "https://github.com/xenserver/buildroot";
+    rev = "5e9892582c3733416bf8b6ebf9aece8cefa15b28";
+    sha256 = "6ee4288fb715e1315ff0cc7923ec1efe896be176d6697a7d56015b77b4317447";
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  buildInputs = [ nix-template-rpm ];
+
+    #cp ${ ./buildRootInput.nix } $out/usr/share/buildroot/buildRootInput.nix
+  installPhase = ''
+    mkdir -p $out/usr/share/buildroot
+    cp -r SPECS $out/usr/share/buildroot/
+    cp -r SOURCES $out/usr/share/buildroot/
+    cp ${ ./translation.table } $out/usr/share/buildroot/translation.table
+
+    #
+    # generate script to update buildRootInput.nix
```

Comment:
Could you add documentation somewhere that explains what buildRootInput.nix is? Three questions come to mind:
* Is it a file somewhere in the nixpkgs repository?
* What does it build?
* What does it mean to update it? (Does it fetch the latest version from some website?)